test/files/jvm/future-spec/FutureTests.scala

```diff
@@ -60,14 +60,14 @@ class FutureTests extends MinimalScalaTest {
       val f3 = f2 map { s => s.toUpperCase }
 
       f2 foreach { _ => throw new ThrowableTest("dispatcher foreach") }
-      f2 onSuccess { case _ => throw new ThrowableTest("dispatcher receive") }
+      f2 onComplete { case Success(_) => throw new ThrowableTest("dispatcher receive"); case _ => }
```

Comment:
@viktorklang You mentioned that `foreach` would make more sense than `onComplete` here. If that's the case, since the previous line already tests `foreach`, should this line just be removed? I don't know if the original intent was to check if `onSuccess` worked, and if it makes sense to test if `onComplete` works as well.

Reply:
@NthPortal I think you can safely assume Viktor was just reading through the diffs quickly and making style suggestions without realizing that this is in a special testing context

Reply:
ok